### PR TITLE
add deck internal job url prefix

### PR DESCRIPTION
**Pull request by droslean**
/cc @petr-muller

### CODE REVIEW

File: core-services/prow/02_config/_config.yaml [hunk 1]
```diff
@@ -345,6 +345,7 @@ plank:
       gcs_credentials_secret: gce-sa-credentials-gcs-private
   job_url_prefix_config:
     '*': https://prow.svc.ci.openshift.org/view/gcs/
+    openshift/private-ci-testing: https://deck-internal-ci.svc.ci.openshift.org/view/gcs/
```
Comment (petr-muller) [inline core-services/prow/02_config/_config.yaml]: Does the config allow to have orgs here, too? If so, we should add `openshift-priv` here.

Comment (droslean) [reply core-services/prow/02_config/_config.yaml]: Yea, since we are doing this, let's add this.

Comment (droslean) [reply core-services/prow/02_config/_config.yaml]: addressed

**Comment by droslean** [comment]
@petr-muller Yes I am planning to fix all of those. But I have to start with the obvious one and clean up the sittuation a bit.

**Pull request merged**